core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1155,6 +1161,9 @@ object KafkaConfig {
       .define(ConsumerGroupMaxSizeProp, INT, Defaults.CONSUMER_GROUP_MAX_SIZE, atLeast(1), MEDIUM, ConsumerGroupMaxSizeDoc)
       .define(ConsumerGroupAssignorsProp, LIST, Defaults.CONSUMER_GROUP_ASSIGNORS, null, MEDIUM, ConsumerGroupAssignorsDoc)
 
+      /** Share Group Configurations **/
+      .defineInternal(ShareGroupEnableProp, BOOLEAN, Defaults.SHARE_GROUP_ENABLE, null, MEDIUM, ShareGroupEnableDoc)
```

Comment:
Agreed. I had initially put the configuration in both server.properties and broker.properties and thought it would be internal but Andrew pointed out that it shouldn't be the case. I'll make the change